config/routes.rb

```diff
@@ -1,7 +1,17 @@
 Rails.application.routes.draw do
-  root "auth#signup"
-
   post "/sessions", to: "auth#login"
   post "/users", to: "auth#signup"
   delete "/sessions", to: "auth#logout"
+
+  post '/hideout', to: "hideouts#create"
+  delete 'hideout', to: "hideouts#destroy"
+  put 'hideout/rename', to: "hideouts#rename"
+  put "/hideout/users", to: "hideouts#add"
+  delete "/hideout/users", to: "hideouts#leave"
+  delete "/hideout/destroy", to: "hideouts#destroy"
```

Comment:
It's not the `to` part that needs pluralization. That can stay as `hideout#<action>`. It's the route URL that should be plural. Like POST `/hideouts` etc.

Reply:
Alright fixed